RegularApi/Domain/Views/DeploymentTemplateView.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace RegularApi.Domain.Views
+{
+    public class DeploymentTemplateView
+    {
+        [Required]
+        public string Name { get; set; }
+        
+        [Required]
+        public string ApplicationId { get; set; }
+        
+        [Required]
+        public HostSetupView HostsSetup { get; set; }
```

Comment:
This should be `IList<HostSetupView>` and rename to `HostSetupViews` and use `JsonProperty` for deserialization :)

Reply:
It is not necessary to reflect the type in the name of the attribute